cssom-1/Overview.bs

```diff
@@ -2034,6 +2038,40 @@ Note: The serialization of an empty CSS declaration block is the empty string.
 Note: The serialization of a non-empty CSS declaration block does not include any surrounding whitespace, i.e., no whitespace appears
 before the first property name and no whitespace appears after the final semicolon delimiter that follows the last property value.
 
+A <a>CSS declaration block</a> has these <a>attribute change steps</a> for its <a for="CSSStyleDeclaration">owner node</a>:
+
+<ol>
+ <li>If the <a for="CSSStyleDeclaration">readonly flag</a> is set, terminate these steps.
+ <li>If the <a for="CSSStyleDeclaration">updating flag</a> is set, terminate these steps.
+ <li>If <var>localName</var> is not "<code>style</code>", or <var>namespace</var> is not null, terminate these steps.
+ <li>If <var>value</var> is null, empty the <a for="CSSStyleDeclaration">declarations</a>.
+ <li>Otherwise, let the <a for="CSSStyleDeclaration">declarations</a> be the result of <a>parse a CSS declaration block</a>
+ from a string <var>value</var>.
+</ol>
+
+When a <a>CSS declaration block</a> object is created, then:
+
+<ol>
+ <li>Let <var>owner node</var> be the <a for="CSSStyleDeclaration">owner node</a>.
+ <li>If <var>owner node</var> is null, or the <a for="CSSStyleDeclaration">readonly flag</a> is set, terminate these steps.
+ <li>Let <var>value</var> be the result of <a lt="get an attribute by namespace and local name">getting an attribute</a>
+ given null, "<code>style</code>", and <var>owner node</var>.
+ <li>If <var>value</var> is not null, let the <a for="CSSStyleDeclaration">declarations</a> be the result of
+ <a>parse a CSS declaration block</a> from a string <var>value</var>.
+</ol>
+
+To <dfn>update style attribute for</dfn> <var>declaration block</var> means to run the steps below:
+
+<ol>
+ <li>Assert: <var>declaration block</var>'s <a for="CSSStyleDeclaration">readonly flag</a> is unset.
+ <li>Let <var>owner node</var> be <var>declaration block</var>'s <a for="CSSStyleDeclaration">owner node</a>.
+ <li>If <var>owner node</var> is null, terminate these steps.
+ <li>Set <var>declaration block</var>'s <a for="CSSStyleDeclaration">updating flag</a>.
+ <li><a>Set an attribute value</a> for <var>owner node</var> using "<code>style</code>" and the result of
```

Comment:
This doesn't handle mutation events right, but I guess neither does anything else....